cms/djangoapps/contentstore/views/course.py

```diff
@@ -67,7 +69,13 @@
 from course_action_state.managers import CourseActionStateItemNotFoundError
 from microsite_configuration import microsite
 from xmodule.course_module import CourseFields
+from xmodule.split_test_module import get_split_user_partitions
 
+MINIMUM_GROUP_ID = 100
+
+CONTENT_GROUP_CONFIGURATION_DESCRIPTION = 'The groups in this configuration can be mapped to cohort groups in the LMS.'
```

Comment:
Maybe add short comment above these explaining why they aren't translated (not currently visible to users). Else someone else is likely to think that this is an oversight in the code.